js/main.js

```diff
@@ -25,37 +26,75 @@ var $newEntryForm = document.querySelector('#entry-form');
 var $entryList = document.querySelector('.entry-list');
 var $entry;
 
-function changeViewDOM() {
+function changeViewEntries() {
   data.view = 'entries';
   $dataViewEntries.className = 'view';
   $dataViewEntryForm.className = 'view hidden';
 }
 
+function changeViewForm() {
+  data.view = 'entry-form';
+  $dataViewEntries.className = 'view hidden';
+  $dataViewEntryForm.className = 'view';
+}
+
+var titleValue = $newEntryForm.elements.title.value;
+var notesValue = $newEntryForm.elements.notes.value;
+var imageValue = $newEntryForm.elements.url.value;
+var entryObj = {
+  url: imageValue,
+  title: titleValue,
+  notes: notesValue,
+  entryId: data.nextEntryId
+};
+
 function clickSave(event) {
-  event.preventDefault();
-  var titleValue = $newEntryForm.elements.title.value;
-  var notesValue = $newEntryForm.elements.notes.value;
-  var imageValue = $newEntryForm.elements.url.value;
-  var entryObj = {
-    url: imageValue,
-    title: titleValue,
-    notes: notesValue,
-    entryId: data.nextEntryId
-  };
-  data.nextEntryId++;
-  data.entries.unshift(entryObj);
-  $entry = renderEntry(entryObj);
-  $entryList.prepend($entry);
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
-  $newEntryForm.reset();
-  changeViewDOM();
+  if ($formHead.textContent === 'New Entry') {
+    titleValue = $newEntryForm.elements.title.value;
+    notesValue = $newEntryForm.elements.notes.value;
+    imageValue = $newEntryForm.elements.url.value;
+    entryObj = {
+      url: imageValue,
+      title: titleValue,
+      notes: notesValue,
+      entryId: data.nextEntryId
+    };
+    event.preventDefault();
+    data.nextEntryId++;
+    data.entries.unshift(entryObj);
+    $entry = renderEntry(entryObj);
+    $entryList.prepend($entry);
+    $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+    $newEntryForm.reset();
+    changeViewEntries();
+  } else {
+    event.preventDefault();
```

Comment:
Which will remove the need for this line

Reply:
Done.